Exercise3/modules/ec2/main.tf

```diff
@@ -0,0 +1,79 @@
+locals {
+  owner = "${var.client_name}-nclouds"
+}
+
+resource "aws_key_pair" "rbtkey" {
+  key_name   = "rishav_o"
+  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAEFD4qFzU8HOcDUSZGbNlej/qmFDIeMHSvrIf6MQG6TDyZrTfWDfypwWywrAtudcfmz8T4We5NOeqkchLqqNZBU41BH91FQEr/sL/f2tmLb9vMBuM1savnXAVCRoH/TtrzISSAm0TWXFKK00TQK4O/QhYMml6A9CsMw12qYO5Mjn1lUiL5SILFtQI2KD8lFX9I2jDSIoYsYyY0fkJYGt/zDBbGlvrCtEWRq5lCEfKjHVHWDpI4393fZoIk6J0uNI2sOraj3D5Q/E7dJyERz38mMArgu+/TDJuP0jSaUO5D48EmdFPDtA2sFX/Z9pwWPpN12fl/HG+4gnEKXtnBrax"
```

Comment:
To make it look a bit cleaner you can try to read the public key file directly by using the file() function
https://developer.hashicorp.com/terraform/language/functions/file